forc-plugins/forc-client/src/cmd/call.rs

```diff
@@ -0,0 +1,241 @@
+use crate::NodeTarget;
+use clap::Parser;
+use either::Either;
+use fuel_crypto::SecretKey;
+use fuels::programs::calls::CallParameters;
+use fuels_core::types::{AssetId, ContractId};
+use std::{path::PathBuf, str::FromStr};
+use url::Url;
+
+pub use forc::cli::shared::{BuildOutput, BuildProfile, Minify, Pkg, Print};
+pub use forc_tx::{Gas, Maturity};
+
+#[derive(Debug, Clone)]
+pub enum FuncType {
+    Selector(String),
+    // TODO: add support for function signatures - without ABI files
```

Comment:
Have we got a github issue for this that we can mention here?

Reply:
No; i will add

Reply:
https://github.com/FuelLabs/sway/pull/6791/commits/e7e0c41d88d6e45698ce37e2b04cd74dc7673392